pom.xml

```diff
@@ -696,9 +718,14 @@
       <dependency>
         <groupId>com.sun.jersey</groupId>
         <artifactId>jersey-client</artifactId>
-        <version>1.19.2</version>
+        <version>1.9</version>
```

Comment:
do all our apis/consoles still work with this change?

Reply:
So `jersey-client` is marked as `test` and will only be used in the tests. I've already verified it.